CI/kuiper/build_scopy_kuiper.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+set -e
+set -x
+export PS4='+(${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+
+HOME="/home/analog"
+STAGING_AREA=$HOME"/staging"
+STAGING_AREA_DEPS=$STAGING_AREA"/dependencies"
+JOBS=1
+
+mkdir -p $STAGING_AREA_DEPS
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$STAGING_AREA_DEPS/lib
+
+apt-get install libsndfile-dev -y
```

Comment:
Could we add a check for a boost version above 1.69, such as "apt-get install libboost1.71-all-dev"

Reply:
Ok, I have added the check